app/src/main/java/com/nutomic/syncthingandroid/service/RunConditionMonitor.java

```diff
@@ -298,13 +294,8 @@ private boolean wifiWhitelistConditionMet(boolean prefWifiWhitelistEnabled,
      * Functions for run condition information retrieval.
      */
     private boolean isCharging() {
-        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
-            // API level < 21
-            return isCharging_API16();
-        } else {
-            // API level >= 21
-            return isCharging_API17();
-        }
+        // API level >= 21
+        return isCharging_API17();
     }
 
     @TargetApi(16)
```

Comment:
Looks like there's something that can go too then? :)

Reply:
👍 , updated the PR 
I removed the code in the method below (annotated with TargetApi 16) and moved the the code from `isCharging_API17()` method directly to `isCharging()` one.